SocialFilter/sections/css_extended.txt

```diff
@@ -2,6 +2,7 @@
 ! Extended CSS rules
 !
 elitedatascience.com#?#.op-footer .col-sm-3 > #custom_html-2 > h2:contains(/^Share$/)
+confidenceman-movie.com#?#.share ul > .share_t:contains(Share)
```

Comment:
Missed `li` tag  and why not `confidenceman-movie.com#?#.share > ul > li.share_t:contains(Share)`?